Adyen/UI/Styles/FormComponentStyle.swift

```diff
@@ -101,6 +108,7 @@ public struct FormComponentStyle: TintableStyle {
         self.secondaryButtonItem = secondaryButton
         self.hintLabel = helper
         self.sectionHeader = sectionHeader
+        self.addressStyle = .init(title: sectionHeader, textField: textField)
```

Comment:
Would it be better to have this value as lazy initialized var than to repeat this a few times?

Reply:
Moving this to a lazy var would lead to breaking changes unfortunately